### DISCUSSION TITLE
[Documentation]: Docs and code out of sync on go2rtc version used

### DISCUSSION THREAD

**Discussion by leccelecce** [blakeblackshear/frigate#14751] [General Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

https://github.com/blakeblackshear/frigate/commit/c72d30451562d6037be2b7d7f807c05e51287eed updated go2rtc from 1.9.2 to 1.9.4 and updated all docs

https://github.com/blakeblackshear/frigate/commit/00e235867a01f053e3c760a2907ad118d2c738b9 downgraded back to 1.9.2 but in the build files only, not in the docs, so they still state 1.9.4.

Haven't submitted a PR to correct in case you'd rather correct by bumping to latest v1.9.6 https://github.com/AlexxIT/go2rtc/releases/tag/v1.9.6

### Steps to reproduce

n/a

### Version

dev

### Frigate config file

```yaml
n/a
```

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

n/a

### Screenshots of the Frigate UI's System metrics pages

n/a

**Comment by hawkeye217** [comment]
There are known issues with 1.9.4 and later which is why we are still on 1.9.2. Most of the docs references to the wrong version are links, so this is pretty minor. But feel free to make a PR to update the docs to 1.9.2.

**Reply by leccelecce** [reply, marked as answer]
PR sent: https://github.com/blakeblackshear/frigate/pull/14753
